src/commands/distribute/release.ts

```diff
@@ -0,0 +1,224 @@
+        out.text("Failed to abort release upload");
+      }
+
+      throw error;
+    }
+
+    debug("Extracting release ID from the release URL");
+    const releaseId = this.extractReleaseId(releaseUrl);
+
+    debug("Distributing the release");
+    const releaseDetails = await this.distributeRelease(client, app, releaseId, releaseNotesString);    
+
+    out.text(`Release ${releaseDetails.shortVersion} (${releaseDetails.id}) was successfully released to ${distributionGroupUsers.length} testers of ${this.distributionGroup}`);
+
+    return success();
+  }
+
+  private validateParameters(): void {
+    if (_.isNil(this.releaseNotes) && _.isNil(this.releaseNotesFile)) {
+      throw failure(ErrorCodes.InvalidParameter, "specify either '--release-notes' or '--release-notes-file' switch");
+    } else if (!_.isNil(this.releaseNotes) && !_.isNil(this.releaseNotesFile)) {
+      throw failure(ErrorCodes.InvalidParameter, "'--release-notes' and '--release-notes-file' switches are mutually exclusive");
+    }
+  }
+
+  private getPrerequisites(client: MobileCenterClient): Promise<[models.DistributionGroupUserGetResponse[], Buffer, string]> {
+    // load release binary file
+    const fileBuffer = Pfs.readFile(this.filePath).catch((error: NodeJS.ErrnoException) => {
+      if (error.code === "ENOENT") {
+        throw failure(ErrorCodes.InvalidParameter, `binary file '${this.filePath}' doesn't exist`);
+      } else {
+        throw error;
+      }
+    });
+
+    // load release notes file or use provided release notes if none was specified
+    const releaseNotesString = !_.isNil(this.releaseNotesFile) ? 
+      Pfs.readFile(this.releaseNotesFile, "utf8").catch((error: NodeJS.ErrnoException) => {
+        if (error.code === "ENOENT") {
+          throw failure(ErrorCodes.InvalidParameter, `release notes file '${this.releaseNotesFile}' doesn't exist`);
+        } else {
+          throw error;
+        }
+      })
+     : Promise.resolve(this.releaseNotes);
+    
+    // get distribution group users (and check distribution group existence)
+    const distributionGroupUsers = this.getDistributionGroupUsers(client);
+
+    return Promise.all([distributionGroupUsers, fileBuffer, releaseNotesString]);
+  }
+
+  private async getDistributionGroupUsers(client: MobileCenterClient): Promise<models.DistributionGroupUserGetResponse[]> {
+    const distributionGroupUsersRequestResponse = await clientRequest<models.DistributionGroupUserGetResponse[]>(
+      (cb) => client.account.getDistributionGroupUsers(this.app.ownerName, this.app.appName, this.distributionGroup, cb));
+    const statusCode = distributionGroupUsersRequestResponse.response.statusCode;
+    if (statusCode >= 400) {
+      if (statusCode === 404) {
+        throw failure(ErrorCodes.InvalidParameter, `distribution group ${this.distributionGroup} was not found`);
+      } else {
+        throw failure(ErrorCodes.Exception, `failed to get users of distribution group ${this.distributionGroup}`);
```

Comment:
I don't think this should fail the command, I would suggest to just update the message to not show the number of testers. Something like this:
"Release 1.0 (45) was successfully released in Beta Testers"